index.html

```diff
@@ -247,6 +277,16 @@ <h4>Skills</h3>
                     <li><a href="#"><img src="images/angellist.svg" alt="Angellist"></a></li>
                 </ul>
             </div>
+
```

Comment:
- [ ] Kindly add the gray border above the icons,  and add a little bit of spacing to the icon.

- Your design
<img width="949" alt="image" src="https://github.com/Wils-FOURR-speed/My-portfolio/assets/41163571/783aa9c6-e08b-4d7c-9c8a-dfc198824db0">

- Figma design
<img width="393" alt="image" src="https://github.com/Wils-FOURR-speed/My-portfolio/assets/41163571/ea3683e1-7571-41ce-926e-cba6cf9c83fa">